pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1435,6 +1435,28 @@ def test_dt64arr_add_sub_offset_ndarray(self, tz_naive_fixture,
         expected = tm.box_expected(expected, box_with_array)
         tm.assert_equal(res, expected)
 
+    @pytest.mark.parametrize("op, offset, exp", [
+        ('__add__', pd.DateOffset(months=3, days=10),
+         DatetimeIndex([Timestamp('2014-04-11'), Timestamp('2015-04-11'),
+                        Timestamp('2016-04-11'), Timestamp('2017-04-11')])),
+        ('__add__', pd.DateOffset(months=3),
+         DatetimeIndex([Timestamp('2014-04-01'), Timestamp('2015-04-01'),
+                        Timestamp('2016-04-01'), Timestamp('2017-04-01')])),
+        ('__sub__', pd.DateOffset(months=3, days=10),
+         DatetimeIndex([Timestamp('2013-09-21'), Timestamp('2014-09-21'),
+                        Timestamp('2015-09-21'), Timestamp('2016-09-21')])),
+        ('__sub__', pd.DateOffset(months=3),
+         DatetimeIndex([Timestamp('2013-10-01'), Timestamp('2014-10-01'),
+                        Timestamp('2015-10-01'), Timestamp('2016-10-01')]))
+
+    ])
```

Comment:
@makbigc I'm a little bit late to the party here, but can you do a follow-up to parametrize over `box` to make sure the add/sub ops work for Series/DataFrame?  Also tzaware and tz-naive.